frontends/web/src/locales/en/app.json

```diff
@@ -793,6 +793,10 @@
         "text": "As soon as you transact, a new address is automatically added to the list so there are always 20 addresses available which have never received any coins.",
         "title": "When do the addresses change?"
       },
+      "addressFormats": {
+        "text": "A compatible address is standard segwit whereas the modern address is native segwit (bech32). By default the BitBox02 uses the modern address. If you are having issues sending to this address, click \"change to compatible address\".",
```

Comment:
Nit: I believe we write Segwit with capital S, most of the time, but there are a few occurences of segwit lowercase.

Reply:
should probably be lowercase everywhere else?

Reply:
Segwit is a name like Bitcoin always capital first letter IMO